public/components/organisms/footer.html

```diff
@@ -1,85 +1,163 @@
-        <ul class="footer_social-icons">
-          <li><a href="http://www.facebook.com"><img src='../../img/icon_facebook.png' /></a></li>
-          <li><a href="http://www.twitter.com"><img src='../../img/icon_instagram.png' /></a></li>
-          <li><a href="http://www.youtube.com"><img src='../../img/icon_twitter.png' /></a></li>
+    </div>
+
+    <div>
+      <hr class="hr-2" />
+    </div>
+
+    <div class="footer-main-2">
+      <h2>Engage</h2>
+
+      <div>
+        <ul class="main_column">
+          <h3>Participants</h3>
+          <li>
+            <a href="#">Student Application</a>
+          </li>
+        </ul>
+      </div>
+
+      <div>
+        <ul class="main_column">
+          <h3>Volunteers</h3>
+          <li>
+            <a href="#">Volunteer Opportunities </a>
+          </li>
+          <li>
+            <a href="#">Volunteer Application</a>
+          </li>
+        </ul>
+      </div>
+
+      <div>
+        <ul class="main_column">
+          <h3>Supporters</h3>
+          <li>
+            <a href="#">Join the Refcode Extended Professional Network</a>
+          </li>
+          <li>
+            <a href="#">Subscribe to our Newsletter</a>
+          </li>
+          <li>
+            <a href="#">Donate</a>
+          </li>
         </ul>
       </div>
-    </footer>
-  </markup>
-  <includes>
-     <link href="https://fonts.googleapis.com/css2?family=Lexend+Deca&display=swap" rel="stylesheet">
-  </includes>
+
+      <div>
+        <ul class="main_column">
+          <h3>Businesses</h3>
+          <li>
+            <a href="#">Partner with us</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="footer-main-3">
+      Refcode in partnership with
+      <a class="underline-footer" href="https://tektontraining.org/">
+        Tektan Career Training
+      </a>
+      is a 501(c)3 nonprofit
+    </div>
+  </footer>
 </div>
+<style>
+  /* =================================
+    Base Layout Styles
+  ==================================== */
+  .footer {
+    background: white;
+    color: black;
+  }
+
+  .footer-container {
+    display: grid;
+    column-gap: 40px;
+  }
+
+  .footer-main-1,
+  .footer-main-2 {
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    column-gap: 40px;
+    margin-left: 75px;
+  }
+
+  .footer-main-3 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    font-size: 12px;
+  }
+
+  .underline-footer {
+    text-decoration: underline;
+    margin-left: 5px;
+    margin-right: 5px;
+  }
+
+  li {
+    list-style-type: none;
+    margin: 0 0 12px 0;
+  }
+
+  a {
+    text-decoration: none;
+    color: #444;
+  }
+
+  h2 {
+    font-family: "Lexend Deca", sans-serif;
+  }
+
+  .main_column {
+    font-family: Lato, sans-serif;
+  }
+
+  .hr-1 {
+    width: 100%;
+    height: 1px;
+    background-color: #444;
+  }
+
+  .hr-2 {
+    width: 90%;
+    height: 1px;
+    background-color: #444;
```

Comment:
The browser applies some default styles to the `hr` element that are making the line 3px high instead of just 1px. Looks like it's a border causing it:

![image](https://user-images.githubusercontent.com/1900802/107090000-8a3ad480-67cd-11eb-937d-0da3c8d26f0b.png)

So you can override that user agent style by setting `border: none;` in your class.